spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2529,4 +2529,21 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
         spark.sql("SELECT array_contains((CASE WHEN _2 =_3 THEN array(_4) END), _4) FROM t1"));
     }
   }
+  test("array_remove") {
```

Comment:
nit: there should be a blank line between tests:

```suggestion
  }
  
  test("array_remove") {
```